pythonLibs/ATATools/ata_pulsars.py

```diff
@@ -0,0 +1,345 @@
+import os
+import subprocess
+import argparse
+
+from astropy.coordinates import EarthLocation, SkyCoord, AltAz
+from astropy.time import Time
+from astropy import units as u
+
+import datetime
+import numpy as np
+import pandas as pd
+
+ATA_LAT = 40.8178
+ATA_LON = 121.473
+
+time_string = "%Y-%m-%d %H:%M:%S"
+
+ATA = EarthLocation.of_site("Allen Telescope Array")
+
+'''
+ATA_PULSARS = PULSARS = 
+       ['J1939+2134', 
+        'J1713+0747', 
+        'J2145-0750', 
+        'J1857+0943', 
+        'J1022+1001', 
+        'J1643-1224', 
+        'J1744-1134', 
+        'J0613-0200', 
+        'J0837+0610', 
+        'J0332+5434', 
+        'J0953+0755', 
+        'J1935+1616', 
+        'J2022+2854', 
+        'J2018+2839', 
+        'J1932+1059', 
+        'J2022+5154', 
+        'J1645-0317', 
+        'J1239+2453', 
+        'J0358+5413']
+'''
+
+ATA_PULSAR_DATA = pd.read_csv("PULSARS.csv")#[el.split(",") for el in f.read().split("\n") if el[0] != '' and if el[0] != "PULSAR"]
```

Comment:
This is an assumption that the `PULSARS.csv` file exists in the current directory. Is that part of the pull request?